Newsletters: add 342 (2025-02-21)

harding opened the pull request:
- [ ] Lede, releases/RCs, topic entries @harding 
- [x] v3 commitments topic @harding 
- [x] Client/services @bitschmidty 
- [x] Notable merges @Gustavojfe

LarryRuane commented on _posts/en/newsletters/2025-02-21-newsletter.md:
```diff
@@ -0,0 +1,155 @@
+  Teinturier starts by outlining the four cases that require a mobile
+  wallet to broadcast a transaction:
+
+  1. Their peer broadcasts a revoked commitment transaction, i.e. the
```
```suggestion
  1. Their peer broadcasts a revoked commitment transaction, i.e., the
```

LarryRuane commented on _posts/en/newsletters/2025-02-21-newsletter.md:
```diff
@@ -0,0 +1,155 @@
+  1. Their peer broadcasts a revoked commitment transaction, i.e. the
+     peer is attempting to steal funds.  In this case, the mobile wallet
+     immediately gains the ability to spend all of the channel's funds,
+     allowing them to use those funds to pay for fees.
```
```suggestion
     allowing it to use those funds to pay for fees.
```

LarryRuane commented on _posts/en/newsletters/2025-02-21-newsletter.md:
```diff
@@ -0,0 +1,155 @@
+  4. The mobile wallet is receiving an HTLC.  In this case, the remote
+     peer can accept the HTLC preimage (allowing it to claim funds from
+     its upstream peers) but not update the settled channel balance and
+     revoke the HTLC.  In this case, the mobile wallet must force close
```
```suggestion
     revoke the HTLC.  In this case, the mobile wallet must force-close
```

LarryRuane commented on _posts/en/newsletters/2025-02-21-newsletter.md:
```diff
@@ -0,0 +1,155 @@
+  allow nodes to signal that one of their channels was highly available
+  (HA)---able to forward payments up to a specified amount with 100%
+  reliability (see [Newsletter #239][news239 qos]).  If a spender
+  chooses a HA channel and the payment fails at that channel, then the
```
```suggestion
  chooses an HA channel and the payment fails at that channel, then the
```

bitschmidty commented on _posts/en/newsletters/2025-02-21-newsletter.md:
```diff
@@ -0,0 +1,222 @@
+  caused confusing and undocumented side effects, but now it simply clears the
+  specified settings to restore the default behaviour.
+
+- [Bitcoin Core #31384][] resolves an issue where the 4,000 weight units (WU)
```
Related news item: https://bitcoinops.org/en/newsletters/2025/01/10/#investigating-mining-pool-behavior-before-fixing-a-bitcoin-core-bug

Gustavojfe replied:
added

bitschmidty commented on _posts/en/newsletters/2025-02-21-newsletter.md:
```diff
@@ -0,0 +1,222 @@
+  channels][topic simple taproot channels]. Changes are made to [BOLT2][] and
+  [BOLT3][].
+
+{% include snippets/recap-ad.md when="2025-02-25 15:30" %}
```
```suggestion
{% include snippets/recap-ad.md when="2025-02-25 17:30" %}
```

bitschmidty commented on _topics/en/v3-commitments.md:
```diff
@@ -0,0 +1,127 @@
+  **V3 commitments** are LN commitment transactions made using version 3
+  transactions adhering to the policies for TRUC, a P2A outputs,
+  ephemeral dust, and sibling replacement.  This allows it to be a
+  **zero-fee commitment**, as the onchain fee will be paid by CPFP spend
```
```suggestion
  **zero-fee commitment**, as the onchain fee will be paid by a CPFP spend
```

bitschmidty commented on _topics/en/v3-commitments.md:
```diff
@@ -0,0 +1,127 @@
+commitment transaction to allow either party to [CPFP][topic cpfp] fee
+bump it if necessary.  This allowed reducing the commitment transaction
+fee to the minimum necessary to get it to relay.  Although this was a
+major improvement, it was still fragile if the minimum relay feerate
```
Could link to Default minimum transaction relay feerates topic here

bitschmidty commented on _topics/en/v3-commitments.md:
```diff
@@ -0,0 +1,127 @@
+
+- [Package relay][topic package relay] allows the commitment transaction
+  to pay zero fee.  This eliminates the fragility of increases in
+  feerates (or minimum relay feerates) making a previously signed
```
Or Default minimum transaction relay feerates here

bitschmidty commented on _topics/en/v3-commitments.md:
```diff
@@ -0,0 +1,127 @@
+  allows either party to increase the fee bump even if the other party
+  created the initial fee bump.
+
+- [Ephemeral dust][topic ephemeral anchors] allows the P2A output have a
```
```suggestion
- [Ephemeral dust][topic ephemeral anchors] allows the P2A output to have a
```

bitschmidty reviewed (commented):
Some comments on v3 commitment topic.

Amazing how much foundational knowledge (and topic references) are required to properly illustrate this topic, well done @harding !

azuchi commented on _posts/en/newsletters/2025-02-21-newsletter.md:
```diff
@@ -0,0 +1,282 @@
+  database. While extending the `dumptxoutset` RPC itself to output a SQLite3
+  database was considered, it was ultimately rejected due to the increased
+  maintenance burden. The script adds no additional dependencies and the
+  resulting database is about twice the size of the compact serialised UTXO set.
```
```suggestion
  resulting database is about twice the size of the compact serialized UTXO set.
```

jirijakes commented on _posts/en/newsletters/2025-02-21-newsletter.md:
```diff
@@ -0,0 +1,282 @@
+layout: newsletter
+lang: en
+---
+This week's newsletter describes an ideal for allowing mobile wallets to
```
Is this a typo or it was meant like this?

```suggestion
This week's newsletter describes an idea for allowing mobile wallets to
```

harding replied:
Typo, thanks!  Fixing.

harding commented:
Made edits for all review comments, thanks @LarryRuane @bitschmidty @azuchi @jirijakes!  Added lede, dropped releases/RCs due to none available, added topic links.  Reviewed @bitschmidty and @Gustavojfe sections (thanks!).

Thanks everyone!

bitschmidty reviewed (commented):
ACK topics and lede, squashed, thank you all 🚀
